### ISSUE TITLE
Determine Premine Allocation Process

### ISSUE THREAD

**Issue by Flaxscrip** [cryptotechguru/Cryptonomicon#37]
[#32](https://github.com/cryptotechguru/Cryptonomicon/issues/32) establishes a size for the premine. 

We need to determine a **process** to allocate the premine in a way that is fair to past, present, and future supporters and needs of the network.

**Comment by macterra** [comment]
x-post from Discord...

I'm thinking we should keep the ledger as a controlled document in Cryptonomicon so any changes go through a review and voting process. A player could make a claim with supporting documentation. We could discuss, maybe ask for more, or whatever and the group decides if it is sufficient.

I'm guessing bigger claims will require more evidence. Maybe small claims require none because no one cares.

**Comment by Flaxscrip** [comment]
We have an established premine. We need to define a clear set of rules by which we will review the requests. Some that were proposed:

1- Evidence of transaction to the original ICO BTC address (address TDB)
2- SAFT paperwork should be recognizable 
3- Others?

It would be important to provide some level of privacy. Maybe we can create a new Role

**Comment by Flaxscrip** [comment]
We also need to consider whether we need premined coins for future purposes (exchanges, bounties, etc).

**Comment by Kinetic-311** [comment]
Since the EQB website does not allow sending of coins off the website, one could provide their login information to verify the token balance before the site is taken down.  One person would be given the information like Christian for verification.

**Comment by macterra** [comment]
Just for the record, screencaps are dead simple to fake. 

![image](https://user-images.githubusercontent.com/434644/57137708-ab5fcd00-6d7e-11e9-9f6b-6a4504bc88f3.png)

Here I added 100k to my EQB balance...
![image](https://user-images.githubusercontent.com/434644/57137695-a0a53800-6d7e-11e9-93d3-6dcd78c3fdac.png)

Also check out my generous signing bonus! :)

**Comment by macterra** [comment]
The best proof is holding mainnet EQB, but without a block explorer how could you prove you owned the address that contains a balance?

**Comment by Kinetic-311** [comment]
Right. I think if we have a list of people that need further verification and provide logins tot he current EQB site that would be the best method at this point. We should also do it as soon as possible to grab the data points because we have no control over the EQB Infrastructure and web servers.
